### DISCUSSION TITLE
Allow to control the margins in WooCommerce Notices widget

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#25711] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi @avivu 
Since you are mostly working for the pro version, I wanted to share a simple but important thing with you...

If you add the WooCommerce Notices widget, it has a 32px margin at the bottom by default. So we can't remove this margin unless using custom css.

![woo notice](https://user-images.githubusercontent.com/88034902/213910918-f0471816-0ed8-41a6-9cfc-d4c0e7b4e81b.png)

![Screenshot (995)](https://user-images.githubusercontent.com/88034902/213911039-337938e5-528d-482a-aea0-cf04f6591bff.png)

### Describe the solution you'd like

Please remove this extra margin. If needed, we can easily add a margin to this widget.

**Comment by nicholaszein** [comment]
Hi there,

### ✈️ This Feature Request has been migrated from issue #21090 

if you still think this request should be considered for our roadmap, please cast your **⬆️ Upvote** on the first comment.

Happy voting! 🙂

**Comment by nicholaszein** [comment]
This Request was mentioned in the following issues and discussions:

- #18275
- #18649
